lib/src/Uncertainty/Algorithm/Analytical/openturns/Analytical.hxx

```diff
@@ -103,12 +105,11 @@ protected:
   friend class Factory<Analytical>;
 
 private:
-
   OptimizationAlgorithm nearestPointAlgorithm_;
   RandomVector event_;
   Point physicalStartingPoint_;
   AnalyticalResult result_;
-
+  
```

Comment:
Same (use black on your Python files to automatically remove this kind of things)